### PR TITLE
DA-3175: Change gerrit reviews to gerrit review comments

### PR DESCRIPTION
**Pull request by ajinkyan83**

### CODE REVIEW

**Comment by lukaszgryglicki** [comment]
Two questions:
1. Why changing reviews to comments, instead of just adding comments. Do we no longer need reviews? I think the review is the most important thing in Gerrit...
2. Did you made a change in all files? I remember 4 files need to be changed when adding removing anything: they are `affiliation/service.go`, `elastic/service.go`, `shared/service.go` (for example [here](https://github.com/LF-Engineering/dev-analytics-affiliation/blob/master/shared/service.go#L126)) and [swagger/dev-analytics-affiliation.yaml](https://github.com/LF-Engineering/dev-analytics-affiliation/blob/master/swagger/dev-analytics-affiliation.yaml#L2434).

**Comment by ajinkyan83** [comment]
For 1 - I have raised a question in the Jira ticket comment related to the values I see. I will wait for the confirmation/inputs. 
For 2 - Done.

**Comment by lukaszgryglicki** [comment]
OK, ping me when you have it resolved, ideally on slack, so I can react ASAP.

**Comment by ajinkyan83** [comment]
@lukaszgryglicki   regarding your question 1) previous comment - I have discussed and confirmed with @1010sachin in the standup today. We need to replace Gerrit reviews with Gerrit comments.

removed wip label and it is ready for review.

File: shared/service.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ var (
 				{Key: "gerrit_approvals", Name: "Approvals"},
 				{Key: "gerrit_changesets", Name: "Active Changesets"},
 				{Key: "gerrit_merged_changesets", Name: "Merged Changesets"},
-				{Key: "gerrit_reviews", Name: "Reviews"},
+				{Key: "gerrit_comments", Name: "Comments"},
```
Comment (1010sachin) [inline shared/service.go:126]: lets call this ```Review Comments```

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Comment by lukaszgryglicki** [comment]
Deployed.

**Pull request merged**